src/inmanta/agent/executor.py

```diff
@@ -317,6 +340,13 @@ def __init__(self, env_path: str, io_threadpool: ThreadPoolExecutor):
         self.inmanta_venv_status_file: pathlib.Path = pathlib.Path(self.env_path) / const.INMANTA_VENV_STATUS_FILENAME
         self.folder_name: str = pathlib.Path(self.env_path).name
         self.io_threadpool = io_threadpool
+        self.constraint_file: pathlib.Path | None = None
+
+    def _write_constraint_file(self, blueprint: EnvBlueprint) -> None:
+        if blueprint.constraints_file_hash is not None and blueprint.constraints:
+            self.constraint_file = pathlib.Path(self.env_path) / blueprint.constraints_file_hash
```

Comment:
Why use the hash as the name. People that look at the venv files will have no clue what this file is for.

Reply:
Great point -> changed to `requirements.txt`